### ISSUE TITLE
Export Windows: Rightclick on Taskbar icon of exported game says "Godot Engine"

### ISSUE THREAD

**Issue by golddotasksquestions** [godotengine/godot#36629]
**Godot version:** 3.2.stable

**OS/device including version:** Win 64

**Issue description:**
Right clicking on the Taskbar Icon of the exported game says "Godot Engine" instead of the game name. 
Hovering over the taskbar icon correctly shows the Project Name (as set in the Project Settings), equally correct is the name in the window when the game does not run in fullscreen mode.

**Steps to reproduce:**
Export game to Windows, right click on the taskbar icon.

**Comment by akien-mga** [comment]
Did you try to change the "Product Name" in the export preset (requires rcedit like for the icon: https://docs.godotengine.org/en/3.2/getting_started/workflow/export/changing_application_icon_for_windows.html)?

**Comment by golddotasksquestions** [comment]
Yes I did that, as I stated. I'm surprised you did not notice, this must be my shortest issue ever ;)
I also have rcedit path asigned. The icon shows up in the taskbar without problem.

**Comment by golddotasksquestions** [comment]
So after a restart, I tested the exported project again, turns out "Godot Engine" is now gone, instead what shows up is the File Description I set in the Windows Export Settings under Application.

**Comment by akien-mga** [comment]
The information you've viewing with a right click on the taskbar icon is determined by the Windows PE file format, it's not for us to define.

If it shows the FileDescription field (which is one of the available strings, see section "string names" in https://docs.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource#remarks), then it's the one you need to set to customize this field.

The default `FileDescription` value is indeed "Godot Engine", it's hardcoded in the binary by the Windows Resource file (`platform/windows/godot_res.rc`) and that's why rcedit is necessary to change it.

**Comment by golddotasksquestions** [comment]
Thanks for the info! Is FileDescription then the right name for this field? Every other software I run just shows the software name when I rightclick the taskbar icon.

If it says "File Description", someone who would want to export to Windows but did not know about what you just explained, would most likely feel inclined to write more than the name of the project, maybe even a small paragraph.

**Comment by akien-mga** [comment]
Yeah it's a bad name but that's the Windows API, we can't change it.

This could be clarified in tooltips, but we don't have a convenient system to add descriptions to export preset properties for now (contrarily to Project Settings).

So in the meantime this should be documented in the Export docs: https://docs.godotengine.org/en/3.2/getting_started/workflow/export/index.html

**Comment by golddotasksquestions** [comment]
I am of course aware that the Godot team does not work for Microsoft ;)
What I meant is the wording as it appears in the Godot Engine. 

![export_description](https://user-images.githubusercontent.com/47016402/75592776-8235c900-5a83-11ea-9d8f-ef01996d9907.png)

Or are those auto generated from the application api?

**Comment by akien-mga** [comment]
That's the name of the Microsoft string. It would be wrong that we change it as it would then be harder for users to know what it corresponds to. You can find documentation online on how Windows handles it's PE data "File Description", but not for any custom name we might chose that fits a particular use of this field in Windows 10 taskbar right click.

**Comment by KoBeWi** [comment]
Any screenshot where it's bugged?
I can't reproduce it in 3.2.4 beta4.

**Comment by akien-mga** [comment]
This was fixed ages ago by adding rcedit to allow customizing RES properties on export, and as of 4.5 it doesn't require rcedit anymore, so it works out of the box.
